### ISSUE TITLE
images are not downloaded

### ISSUE THREAD

**Issue by falkorichter** [Triple-T/gradle-play-publisher#34]
I have this gradle file to manage my upload:
```
apply plugin: 'play'

play {
    serviceAccountEmail = System.getenv("PLAYSTORE_EMAIL")
    pk12File = file(System.getenv("PLAYSTORE_PK12_KEYSTORE"))
    track = 'alpha'
}
```

When I run ```bootstrapShowcaseReleasePlayResources```(I have a flavor called "Showcase") The images are not beeing downloaded.

Am I doing something wrong or is this feature broken? Is there a way I can tell the plugin to be verbose?

**Comment by bhurling** [comment]
For now we can't support image downloading as discussed in #14 and #18

Short version is: The Google API only supports downloading thumbnails so we disabled that feature for now. All we can do is set up the folder structure.

I hope that Google will change their API so we can download the full sized images.

**Comment by bhurling** [comment]
Closed - duplicate of #18
